### DISCUSSION TITLE
Global hotkey to toggle muting/unmuting the currently-active process in Trumpet

### DISCUSSION THREAD

**Discussion by RandomNerd1** [File-New-Project/EarTrumpet#1499] [Ideas]
This is the feature I primarily use this program for, and it works great! It'd be even better if I could have a hotkey do this while I'm in a program/game I want to mute/unmute.

Currently, I have to alt+tab out of a fullscreen game, for example, and then go click the tray icon, then click the process's icon in the flyout, then switch back to my program/game. :)

**Comment by Aresic** [comment]
This is what im looking for. I need an application to allow me muting my games fast with an hotkey while im listening music or watching  youtube and seems they all just focus on audio input/output
